src/NServiceBus.Testing.Fakes/Fakes/OutgoingMessage.cs

```diff
@@ -0,0 +1,31 @@
+namespace NServiceBus.Testing
```

Comment:
@andreasohlund We might put other test-related extensions and utils here, it makes sense to put the fake implementations in their own folder.